metrics-dropwizard/src/main/scala/com/mesosphere/usi/metrics/dropwizard/reporters/DataDogAPIReporter.scala

```diff
@@ -51,31 +54,49 @@ class DataDogAPIReporter(settings: DataDogApiReporterSettings, registry: MetricR
     val buffer = new StringBuilder
     val timestamp = Clock.systemUTC().instant().getEpochSecond
 
-    registry.getGauges.asScala.foreach { case (name, gauge) => reportGauge(buffer, sanitizeName(name), gauge, timestamp) }
-    registry.getCounters.asScala.foreach { case (name, counter) => reportCounter(buffer, sanitizeName(name), counter, timestamp) }
-    registry.getHistograms.asScala.foreach { case (name, histogram) => reportHistogram(buffer, sanitizeName(name), histogram, timestamp) }
-    registry.getMeters.asScala.foreach { case (name, value) => reportMetered(buffer, sanitizeName(name), value, timestamp) }
-    registry.getTimers.asScala.foreach { case (name, value) => reportTimer(buffer, sanitizeName(name), value, timestamp) }
+    registry.getGauges.asScala.foreach {
+      case (name, gauge) =>
+        reportGauge(buffer, sanitizeName(name), gauge, timestamp)
+    }
+    registry.getCounters.asScala.foreach {
+      case (name, counter) =>
+        reportCounter(buffer, sanitizeName(name), counter, timestamp)
+    }
+    registry.getHistograms.asScala.foreach {
+      case (name, histogram) =>
+        reportHistogram(buffer, sanitizeName(name), histogram, timestamp)
+    }
+    registry.getMeters.asScala.foreach {
+      case (name, value) =>
+        reportMetered(buffer, sanitizeName(name), value, timestamp)
+    }
+    registry.getTimers.asScala.foreach {
+      case (name, value) =>
+        reportTimer(buffer, sanitizeName(name), value, timestamp)
+    }
 
     val data = buffer
       .insert(0, "{\"series\":[")
       .append("]}")
       .toString
 
     val body = HttpEntity(MediaTypes.`application/json`, data)
-    http.singleRequest(HttpRequest(method = HttpMethods.POST, uri = apiUrl, entity = body)).pipeTo(self)
+    http
+      .singleRequest(HttpRequest(method = HttpMethods.POST, uri = apiUrl, entity = body))
+      .pipeTo(self)
```

Comment:
this is the same issue we had previous with 
```
Call
     . xyz
```
it is frustrating to look at
much prefer

```
http.singleRequest(HttpRequest(method = HttpMethods.POST, uri = apiUrl, entity = body))
       .pipeTo(self)
```

Reply:
yeah, it does not automatically format this way but it also does not format back. I reverted many places, this one slipped. Thanks